### DISCUSSION TITLE
Do we have to write a lot of specific implementations on Android and iOS?

### DISCUSSION THREAD

**Discussion by JackieBuxiaohui** [cashapp/redwood#2235] [General]
Implement every widget on android or ios ？ Android：TExtView、Button、Recyclerview……  iOS：CollectionView……

**Comment by JakeWharton** [comment]
Yes. The only components we provide are rows and columns, and a lazy column. You will probably have to define the schema for tens or a hundred a hundred widgets, and write the bindings to each of those for every supported UI toolkit you support.

**Reply by JackieBuxiaohui** [reply]
Wow, that's a lot of work. It would be nice if we could do the work  together，众人拾柴火焰高
